README.md

```diff
@@ -100,7 +100,7 @@ Select 1:
 What is the difference between a fork and a clone?
 
 ```text
-Your answer here.
+A 'fork' is a request to clone a project and register it under you username, which allows you to pull from multiple 'forks'. A clone doesn't live publicly on your Github account unless you create a repo manually and push it to that repo.
```

Comment:
Close.  On github, `fork` creates a copy of someone else's repository to your account .  Clone is used to make a copy of anyone's repo, creating a local working directory.